server/api_channel.go

```diff
@@ -103,6 +103,17 @@ func (p *Plugin) handleSince(c *gin.Context) {
 		return
 	}
 
+	p.track(evSummarizeUnreadMessages, map[string]interface{}{
+		"channel_id": channel.Id,
+		"user_id":    user.Id,
```

Comment:
Other parts of telemetry (in other plugins and MM code) use `user_actual_id` for identifying the user. It would be nice to stick to the convention. A more detailed list of conventions is avaiilable [here](https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2766831623/Event+payload).

Reply:
Done! Thanks for the heads up.